daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DarReader.scala

```diff
@@ -90,7 +88,7 @@ object GenDarReader {
     }
 
     private[GenDarReader] def readDalfNames: Try[Dar[String]] =
-      bracket(get(ManifestName).map(_.toInputStream))(is => Try(is.close()))
+      get(ManifestName)
```

Comment:
Does this change the guarantees given to the callers with regards to cleaning up?

Reply:
There is no resource to cleanup. Previously, we opened the stream here and closed it here. Now `DarManifestReader.dalfNames` opens and closes the stream.

Reply:
`get(ManifestName)` returns a immutable value no need for clean up here anymore.
The creation of the `InputStream` (with its cleanup is done) in `DarManifestReader.scala` line 18.